CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ const message = new CallMessage({
    contentType: CallMessage.ContentType.ApplicationJson,
    messageType: CallMessage.MessageType.UserDefinedMessage
 })
-const outgoingCallMessage: OutgoingCallMessage = await call.sendMessage(message)
+const outgoingCallMessage: OutgoingCallMessage = await call.sendMessage(message: CallMessage)
```

Comment:
```suggestion
const outgoingCallMessage: OutgoingCallMessage = await call.sendMessage(message);
```

Reply:
sorry, should reflect this comment now: https://github.com/twilio/twilio-voice-react-native/pull/272#discussion_r1472148568